content/neuvector-prime/azure/contents.lr

```diff
@@ -4,17 +4,17 @@ theme: suse
 ---
 body:
 
-##  How to install NeuVector and configure Billing-adapter.
+##  How to install NeuVector and configure Billing-adapter
 
 The NeuVector offer handles billing by gathering usage data from NeuVector. For complete installation, Ensure to follow all the steps
 till NeuVector Usage Record and Billing.
 
 ##Prerequisites
 
-These instructions assume you have an AKS cluster installed.
+These instructions assume you have an AKS cluster installed as per the [NeuVector System requirements document](https://open-docs.neuvector.com/basics/requirements)
 
 
-A step by step walk-through for creating a new offer from the Marketplace Overview page.
+A step by step walk-through for creating a new offer from the Azure Marketplace Overview page.
```

Comment:
The customer does not create an "offer". I suspect was meant to explain how to complete a new deployment of NeuVector into a running cluster?